safe/definitions/reports/infographic.py

```diff
@@ -0,0 +1,250 @@
+# coding=utf-8
+
+"""Definitions relating to infographic template elements."""
+
+from safe.utilities.i18n import tr
+from safe.utilities.resources import resources_path
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+"""Map elements"""
+
+
+map_overview = {
+    'id': 'map-overview'
+}
+
+
+"""HTML frame elements"""
+
+
+people_section_notes = {
+    'id': 'people-section-notes',
+    'mode': 'text',
+    'text': '',
+    'component': 'infographic-people-section-notes'
+}
+
+population_chart_legend = {
+    'id': 'population-chart-legend',
+    'mode': 'text',
+    'text': '',
+    'component': 'population-chart-legend'
+}
+
+html_frame_elements = [people_section_notes, population_chart_legend]
+
+
+"""Header label elements"""
+
+
+infographic_header = {
+    'id': 'infographic-header',
+    'string_format': tr('Estimated impact of {impact_function_name}')
+}
+
+map_overview_header = {
+    'id': 'map-overview-header',
+    'string_format': tr('Event location')
+}
+
+population_chart_header = {
+    'id': 'population-chart-header',
+    'string_format': tr('Estimated number of people affected by hazard level')
+}
+
+people_section_header = {
+	'id': 'people-section-header',
+	'string_format': tr('People affected by hazard')
+}
+
+age_gender_section_header = {
+    'id': 'age-gender-section-header',
```

Comment:
__[flake8]__
* [E101] indentation contains mixed spaces and tabs

<!-- Comment by [SideCI](https://sideci.com) -->